app/services/katello/pulp3/docker_manifest.rb

```diff
@@ -16,7 +16,8 @@ def self.ids_for_repository(repo_id)
 
       def self.content_unit_list(page_opts)
         page_opts[:media_type] = ['application/vnd.docker.distribution.manifest.v1+json',
-                                  'application/vnd.docker.distribution.manifest.v2+json']
+                                  'application/vnd.docker.distribution.manifest.v2+json',
+                                  'application/vnd.oci.image.manifest.v1+json']
```

Comment:
Should we have more entries  https://docs.pulpproject.org/pulp_container/restapi.html#tag/Content:-Manifests/operation/content_container_manifests_list

Seem to see more media types there

Reply:
The remaining two types relate to manifest lists it appears. One of which is missing from the manifest lists filter... I should add it.